### ISSUE TITLE
Controls for ailerons and elevator not working in "Landing Practice" mode

### ISSUE THREAD

**Issue by u2fly** [YSCEDC/YSCE#6]
- **Platform/OS:** Linux (Debian 11.x)
- **YSFlight version:** `20181124` - AFFECTED
- **YSCE version:** `master` - AFFECTED

### Expected behavior

Control for ailerons and elevator should work as expected in "Landing Practice" mode.

### Actual behavior

In "Landing Practice" mode controls for **ailerons** and **elevator** are NOT working (*stuck?*) — when press assigned shortcuts there are NO any effect.

Other shortcuts (including **rudder**) seems like work as expected.

This bug detected since YSFlight 2017's test versions.

### Screenshots

![Screenshot_2022-09-02_15-29-54](https://user-images.githubusercontent.com/22447924/188144308-acef67cb-b36e-43cc-b0f3-53fdd1ff77cf.png)

**Comment by NodokaHanamura** [comment]
I don't believe we have the resources to fix this just yet, but we'll take a look at it and keep it in mind to patch when we can.

**Comment by NodokaHanamura** [comment]
Reopening this and seeing if Jobbin can take a look at it sometime. @waspe414 - if you feel up for it you can tackle this too as part of your bugfix branch.

**Comment by waspe414** [comment]
Yep, I've copied everything from here and from the Discord into my bug tracker spreadsheet. Will try to solve them all
